docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst

```diff
@@ -840,3 +840,4 @@ Portable Globe and Server 5.2
    :class: inline
 .. |Broadcast globe icon| image:: ../../art/fusion/portable/broadcast_false.gif
    :class: inline line-sized
+

```

Comment:
.. |Broadcast globe icon| image:: ../../art/fusion/portable/broadcast_false.gif
   :width: 38px
   :height: 31px

This icon needs to be fixed in portableUserGuideWinLinux.rst also.

Reply:
committed fix.